main/ry835ai.go

```diff
@@ -408,6 +408,17 @@ func calculateNACp(accuracy float32) uint8 {
 	return ret
 }
 
+/*
+processNMEALine parses NMEA-0183 formatted strings against several message types.
+
+Standard messages supported: RMC GGA VTG GSA
+U-blox proprietary messages: PUBX,00 PUBX,03 PUBX,04
+
+return is false if errors occur during parse, or if GPS position is invalid
```

Comment:
What about the case when there is no position in the line being parsed? Is that an "invalid position"?